src/phoenix/db/models.py

```diff
@@ -538,14 +538,17 @@ class ExperimentRun(Base):
     error: Mapped[Optional[str]]
 
 
-class ExperimentEvaluation(Base):
-    __tablename__ = "experiment_evaluations"
+class ExperimentAnnotation(Base):
+    __tablename__ = "experiment_annotations"
     id: Mapped[int] = mapped_column(primary_key=True)
     experiment_run_id: Mapped[int] = mapped_column(
         ForeignKey("experiment_runs.id", ondelete="CASCADE"),
         index=True,
     )
     name: Mapped[str]
+    annotator_kind: Mapped[str] = mapped_column(
+        CheckConstraint("annotator_kind IN ('LLM', 'HUMAN')", name="valid_annotator_kind"),
```

Comment:
```suggestion
        CheckConstraint("annotator_kind IN ('LLM', 'CODE', 'HUMAN')", name="valid_annotator_kind"),
```

Reply:
addressed in de0df5f60ab4b38672f2e289c599b1f24e8851da